api/src/main/scala/WebServer.scala

```diff
@@ -0,0 +1,125 @@
+package org.wikiwatershed.mmw.geoprocessing
+
+import akka.http.scaladsl.unmarshalling.Unmarshaller._
+import spray.json._
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport._
+import DefaultJsonProtocol._
```

Comment:
Oh. Okay let's keep it then. IntelliJ IDEA usually is right about these things, but every once in a while it'll goof.